go/vt/automation/testing_emit_echo_task.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015, Google Inc. All rights reserved.
```

Comment:
I expressed the testing code nature of this class with the prefix "testing_".

The suffix "_test" may mislead people to think this is a test?